chats.py

```diff
@@ -1,5 +1,6 @@
 from flask import json, jsonify
-
+from datetime import datetime
+import time
```

Comment:
Kur šis tiek izmantots?

Reply:
Tiek izmantots datetime. Time paspēlējos, bet kodā netiek lietots.